src/lib.rs

```diff
@@ -0,0 +1,21 @@
+use lrlex::{lrlex_mod, DefaultLexerTypes, LRNonStreamingLexerDef};
+use lrpar::lrpar_mod;
+
+lrlex_mod!("calc.l");
+lrpar_mod!("calc.y");
+
+pub fn eval(
+    lexerdef: &LRNonStreamingLexerDef<DefaultLexerTypes>,
+    input: &String,
```

Comment:
It's more idiomatic to use `&str` instead of `&String`.

Reply:
done